time="2023-10-24T16:45:06-05:00" level=error msg="pq: column gm.grupo_id does not exist" file=grupo_usecase method=SaveGrupoMessage
time="2023-10-24T16:47:25-05:00" level=error msg="pq: column gm.grupo_id does not exist" file=grupo_usecase method=SaveGrupoMessage
time="2023-10-24T16:49:23-05:00" level=error msg="pq: column gm.parent_id does not exist" file=grupo_usecase method=SaveGrupoMessage
time="2023-10-24T19:08:31-05:00" level=error msg="dial tcp 10.0.0.132:5432: connectex: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond." file=grupo_usecase method=SaveGrupoMessage
time="2023-10-24T19:08:31-05:00" level=error msg="dial tcp 10.0.0.132:5432: connectex: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond." file=chat_usecase method=GetChatUser
time="2023-10-24T19:08:52-05:00" level=error msg="dial tcp 10.0.0.132:5432: connectex: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond." file=grupo_ucase method=UpdateUserGrupoLastTimeUpdateMessage
time="2023-10-24T19:08:53-05:00" level=error msg="dial tcp 10.0.0.132:5432: connectex: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond." file=grupo_usecase method=SaveGrupoMessage
time="2023-10-24T19:08:53-05:00" level=error msg="dial tcp 10.0.0.132:5432: connectex: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond." file=chat_usecase method=GetChatUser
time="2023-10-24T19:09:14-05:00" level=error msg="dial tcp 10.0.0.132:5432: connectex: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond." file=grupo_ucase method=UpdateUserGrupoLastTimeUpdateMessage
time="2023-10-24T19:09:28-05:00" level=error msg="dial tcp 10.0.0.132:5432: connectex: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond." file=grupo_usecase method=SaveGrupoMessage
time="2023-10-24T19:09:28-05:00" level=error msg="dial tcp 10.0.0.132:5432: connectex: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond." file=chat_usecase method=GetChatUser
time="2023-10-24T19:09:49-05:00" level=error msg="dial tcp 10.0.0.132:5432: connectex: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond." file=grupo_ucase method=UpdateUserGrupoLastTimeUpdateMessage
